.travis.yml

```diff
@@ -9,9 +9,16 @@ branches:
   - develop
 
 env:
-  matrix:
-  - BACKEND_TEST=true JS_TEST=false
-  - BACKEND_TEST=false JS_TEST=true
+- RUN_BACKEND_TESTS=true RUN_FRONTEND_TESTS=false REPORT_COVERAGE=false
+- RUN_BACKEND_TESTS=false RUN_FRONTEND_TESTS=true REPORT_COVERAGE=false
```

Comment:
Ah, no. REPORT_COVERAGE only refers to REPORT_BACKEND_COVERAGE.

I can see why this may be misleading though. I'll change the variable name?